tests/foreman/api/test_provisioning.py

```diff
@@ -142,6 +142,11 @@ def test_rhel_pxe_provisioning(
     host = host.read()
     assert host.build_status_label == 'Installed'
 
+    # The label checked above is generated dynamically and may not necessarily
+    # represent the exact value that is stored in the database
+    found = sat.api.Host().search(query={'search': f'name="{host.name}" and build_status = built'})
+    assert len(found) == 1
```

Comment:
```suggestion
    assert len(sat.api.Host().search(query={'search': f'name="{host.name}" and build_status = built'})) == 1
```

Reply:
ruff then suggests formatting it like this
```python
assert (
    len(sat.api.Host().search(query={'search': f'name="{host.name}" and build_status = built'}))
    == 1
)
```

Is that still acceptable? Personally I'm not very fond of it

Reply:
Yeah, that's acceptable. We are already using same at many places across this module.